PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -1300,9 +1303,11 @@ void PMTraceConsumer::HandleDWMEvent(EVENT_RECORD* pEventRecord)
         if (eventIter != mWin32KPresentHistoryTokens.end()) {
             TRACK_PRESENT_PATH(eventIter->second);
             DebugModifyPresent(*eventIter->second);
-            eventIter->second->DwmNotified = true;
```

Comment:
Why'd you delete the setting of `DwmNotified`?

Reply:
Ah, that was a typo, fixed.